PerimeterXModule/Internals/Templates/block.mustache

```diff
@@ -134,9 +134,17 @@
         function (){
         window._pxAppId = '{{appId}}';
         var p = document.getElementsByTagName("script")[0], s = document.createElement("script");
+
         s.async = 1;
-        s.src = '//client.perimeterx.net/{{appId}}/main.min.js';
+        s.src = '{{{jsClientSrc}}}';
```

Comment:
`{{jsClientSrc}}`

Reply:
triple brackets indicate "dont escape"
you can see docs here
https://mustache.github.io/mustache.5.html

Reply:
👍